koku/providers/ocp/ocp_provider.py

```diff
@@ -17,9 +17,13 @@
 """OCP service provider implementation to be used by Koku."""
 import logging
 
+from django.core.exceptions import ObjectDoesNotExist, ValidationError
 from django.utils.translation import ugettext as _
 from rest_framework import serializers
+from tenant_schemas.utils import tenant_context
 
+from api.provider.models import Provider
+from reporting.provider.ocp_aws.models import OCPAWSUsageLineItemDaily
```

Comment:
```suggestion
from reporting.provider.ocp_aws.models import OCPAWSCostLineItemDailySummary
```

Reply:
If you don't mind, I'd like to open a tech debt story to take care of this in a separate PR.  This change basically messes up the testing as it stands now.  I started making the test data updates but decided it might be better to get this in so the UI can integrate and come back for the performance improvements.

Reply:
Sounds good to me.